code/game/objects/items/storage/boxes.dm

```diff
@@ -1296,13 +1296,32 @@
 		/obj/item/stack/sheet/mineral/wood/fifty = 1,
 		/obj/item/stack/sheet/mineral/titanium/fifty = 1,
 		/obj/item/stack/sheet/mineral/uranium/fifty = 1,
-		/obj/item/stack/sheet/mineral/diamond = 1,
+		/obj/item/stack/sheet/mineral/diamond/fifty = 1,
 		/obj/item/stack/sheet/mineral/plasma/fifty = 1,
 		/obj/item/stack/sheet/mineral/gold/fifty = 1,
 		/obj/item/stack/sheet/mineral/silver/fifty = 1,
 		/obj/item/stack/sheet/mineral/bananium/fifty = 1)
 	generate_items_inside(items_inside,src)
 
+/obj/item/storage/box/materials/reduced
+	name = "Materials Box"
+	desc = "Contains most of what you would need"
+
+/obj/item/storage/box/materials/reduced/PopulateContents()
+	var/static/items_inside = list(
+		/obj/item/stack/sheet/metal = 10,
+		/obj/item/stack/sheet/glass = 10,
+		/obj/item/stack/sheet/plastic = 10,
+		/obj/item/stack/sheet/mineral/plastitanium = 10,
```

Comment:
No plastitanium in the ORM, should be removed

Reply:
odd i thought i removed it